src/main/proto/netflix/titus/titus_base.proto

```diff
@@ -301,6 +301,14 @@ message SignedAddressAllocation {
     /// (Required) The IP address and metadata that was allocated.
     AddressAllocation addressAllocation = 1;
 
-    // (Required) This is an x509v3 signed version of the address allocation.
-    bytes signedAddressAllocation = 2;
-};
\ No newline at end of file
+    // authoritative publicKey is the key that came from the database
+    bytes authoritativePublicKey = 3;
+    // the hostPublicKey is the ephemeral public key
+    bytes hostPublicKey = 4;
```

Comment:
I'm wondering why we use more than the VPC service's Metatron keys.

Reply:
The metatron keys / certs cannot be used for signing long term material. They are short-lived and not designed to sign offline material.

Reply:
Where/how do we get/manage the"authoritative" keys? Are they Metatron keys with expiry/lifecycle that we control?

Reply:
They are metatron encrypted JSON files.